main.py

```diff
@@ -0,0 +1,67 @@
+from comet_ml import Experiment
+
+import tensorflow as tf
+import numpy as np
+
+from model import VGG, VGG_Spinal
+from preprocess import readData
+
+from tqdm import tqdm
+
+experiment = Experiment(log_code=True)
+
+'''
+hyper params
+'''
+data_path = 'quickdraw'
+num_epoch = 20
+batch_size = 256
+
+loss_fn = tf.keras.losses.CategoricalCrossentropy(from_logits=True)

```

Comment:
I think from_logitis should be False. https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy
Our predictions will be probability distributions.

Reply:
Updated